.github/workflows/cherry_pick_comment.yml

```diff
@@ -0,0 +1,19 @@
+name: Cherry Pick On Comment
```

Comment:
the expected flow is when a pr is merged, look for the label and then cherry pick to the relevant branch(es).
cherry-pick should be blocked on an open pr

Reply:
>Current code flow is. Once a PR is merged, add a comment with /cherry-pick <branch_name>

this should be enabled not only when a pr is merged; as a user i would like to add the comment even when i just open the pr as the merge may not be done by me.

Reply:
Workflow does not run that way. It is triggered on `types: [created]` of issue_comment. It would run when a user creates a comment. As far as I know there is no queue mechanism here. If we want that, then tide is the best way to proceed.

Reply:
this is why i think it should be in our python

or

on pr merge and check existing labels

Reply:
This should not be in python code (in the same repo). Workflow would need to check out the repo and cherry-pick would again need to checkout a different branch in order to do a cherry-pick. This would be a bad idea to add python code in this repository to handle that.

Reply:
> on pr merge and check existing labels

This does not seem like a good idea to me. If I create a patch today and it gets merged say after a month, should my cherry-pick comment be considered relevant? I feel, it should be user's responsibility to add this cherry-pick comment explicitly when it is needed.

Reply:
> If I create a patch today and it gets merged say after a month, should my cherry-pick comment be considered relevant?

any reason why not? if the code should be cherry-picked. and this is a label so the user sees this label and can remove if they decide this is not needed.
i think it is safer than relying on adding the comment after merge